### PR TITLE
Bump drivers from 1.8.0 to 2021.10.0

### LINKED ISSUE #227: Bump driver versions
We've had another release! Version: 2021.10.0

- [ ] Update the rstudio-docker-products for the new pro-drivers

### PR DESCRIPTION
**Pull request by zfouts**
Bump drivers from 1.8.0 -> 2021.10.0
Fixes #227

### CODE REVIEW

File: r-session-complete/centos7/.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 R_VERSION=4.1.0
 PYTHON_VERSION=3.9.5
-DRIVERS_VERSION=1.8.0-1
+DRIVERS_VERSION=2021.10.0
```
Comment (colearendt) [inline r-session-complete/centos7/.env:3]: Are we sync'd up on version now? CI will tell us if this fails, but I thought CentOS always had a `-1` appended 😞

**Review by colearendt — COMMENTED** [review]
Nice! LGTM! Just one little nit, but things may have changed since I remember last

File: r-session-complete/centos7/.env (hunk 1 above)
Comment (zfouts) [reply r-session-complete/centos7/.env]: Fixed, yes it's a limitation with rpm -- totally overlooked

**Review by colearendt — APPROVED** [review]
🎉

**Pull request merged**